sample-data/Samples/Sample XML/Lakeside-StudentProgram.xml

```diff
@@ -15303,6 +15312,36 @@
 				<EducationOrganizationId>255902</EducationOrganizationId>
 			</EducationOrganizationIdentity>
 		</EducationOrganizationReference>
+		<SchoolFoodServiceProgramService>
+            <SchoolFoodServiceProgramService>uri://ed-fi.org/SchoolFoodServiceProgramServiceDescriptor#Free Lunch</SchoolFoodServiceProgramService>
+        </SchoolFoodServiceProgramService>
+	</StudentSchoolFoodServiceProgramAssociation>
+	<StudentSchoolFoodServiceProgramAssociation>
```

Comment:
Several of these new Lakeside entries didn't load, the equivalent Bear Creek did though.

`    94700 WARN  ApiLoaderApplication - Lakeside-StudentProgram.xml StudentSchoolFoodServiceProgramAssociation Line 15319 Level: 15 #6488 Forbidden - 403 - {
15:39:56       "message": "Access to the resource item could not be authorized because the 'StudentUSI' of the resource is empty."
15:39:56     }
15:39:56     94703 ERROR ApiLoaderApplication - Lakeside-StudentProgram.xml StudentSchoolFoodServiceProgramAssociation Line 15319 Level: 15 #6566 Forbidden - 403 - {
15:39:56       "message": "Access to the resource item could not be authorized because the 'StudentUSI' of the resource is empty."
15:39:56     }
15:39:56     94708 WARN  ApiLoaderApplication - Lakeside-StudentProgram.xml StudentSchoolFoodServiceProgramAssociation Line 15913 Level: 15 #6510 Forbidden - 403 - {
15:39:56       "message": "Access to the resource item could not be authorized because the 'StudentUSI' of the resource is empty."
15:39:56     }
15:39:56     94710 ERROR ApiLoaderApplication - Lakeside-StudentProgram.xml StudentSchoolFoodServiceProgramAssociation Line 15913 Level: 15 #6567 Forbidden - 403 - {
15:39:56       "message": "Access to the resource item could not be authorized because the 'StudentUSI' of the resource is empty."
15:39:56     }
15:39:56     94872 WARN  ApiLoaderApplication - Lakeside-StudentProgram.xml StudentSchoolFoodServiceProgramAssociation Line 17155 Level: 15 #6556 Forbidden - 403 - {
15:39:56       "message": "Access to the resource item could not be authorized because the 'StudentUSI' of the resource is empty."
15:39:56     }
15:39:56     94874 ERROR ApiLoaderApplication - Lakeside-StudentProgram.xml StudentSchoolFoodServiceProgramAssociation Line 17155 Level: 15 #6568 Forbidden - 403 - {
15:39:56       "message": "Access to the resource item could not be authorized because the 'StudentUSI' of the resource is empty."
15:39:56     }
15:39:56     94882 WARN  ApiLoaderApplication - Lakeside-StudentProgram.xml StudentSchoolFoodServiceProgramAssociation Line 17371 Level: 15 #6564 Forbidden - 403 - {
15:39:56       "message": "Access to the resource item could not be authorized because the 'StudentUSI' of the resource is empty."
15:39:56     }
15:39:56     94883 ERROR ApiLoaderApplication - Lakeside-StudentProgram.xml StudentSchoolFoodServiceProgramAssociation Line 17371 Level: 15 #6569 Forbidden - 403 - {
15:39:56       "message": "Access to the resource item could not be authorized because the 'StudentUSI' of the resource is empty."
15:39:56     }`